### ISSUE TITLE
The aria-label attribute is stripped from phone number links by text formatter

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#1957]
**Describe the defect**
The aria-label attribute is stripped from phone number links by text formatter

**To Reproduce**

1. Go to /node/add/page
2. Add a link in a WYSIWYG field like so: `<a href="tel:711" aria-label="TTY. 7 1 1.">711</a>`
3. Save the node
4. Note that the `aria-label` attribute was stripped.

**Expected behavior**
The `aria-label` attribute should be saved in the content.

Related story from Public Websites team: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/8379
